cmd/ltkeycloak/main.go

```diff
@@ -0,0 +1,468 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
```

Comment:
Overall, this file is too big. Suggest moving out `RunSyncFromKeycloakCommandF` and `RunSyncFromMattermostCommandF` to their own files.